doc/content/source/auxkernels/DensityMoles.md

```diff
@@ -1,20 +1,23 @@
 # DensityMoles
 
-!alert construction title=Undocumented Class
-The DensityMoles has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /AuxKernels/DensityMoles
```

Comment:
I can't make this comment on the density moles source code but the class description needs the math environment for the #/m^3 it should be #/m$^3$ so the exponent displays properly